pkg/kv/migration.go

```diff
@@ -39,21 +39,20 @@ func (d *DatabaseMigrator) Migrate(ctx context.Context) error {
 	return nil
 }
 
-func ValidateSchemaVersion(ctx context.Context, store Store, migrationRequired bool) error {
+var errMigrationRequired = errors.New("wrong kv version")
+
+func ValidateSchemaVersion(ctx context.Context, store Store) error {
 	kvVersion, err := GetDBSchemaVersion(ctx, store)
-	if errors.Is(err, ErrNotFound) && !migrationRequired {
-		logging.Default().Info("No KV Schema version, setup required")
-		return nil
+	if errors.Is(err, ErrNotFound) {
+		// probably a new installation
+		return ErrNotFound
 	}
 	if err != nil {
 		return fmt.Errorf("get KV schema version: %w", err)
 	}
-
 	if kvVersion < InitialMigrateVersion {
-		if migrationRequired {
-			return fmt.Errorf("migration to KV required, for more information see https://docs.lakefs.io/reference/upgrade.html#lakefs-0800-or-greater-kv-migration : %w", err)
-		}
-		return fmt.Errorf("(scehma version %d): %w", kvVersion, ErrInvalidSchemaVersion)
+		logging.Default().Info("Migration to KV required. Did you migrate using version v0.80.1? https://docs.lakefs.io/reference/upgrade.html#lakefs-0800-or-greater-kv-migration")
```

Comment:
Suggestion - use v0.80.x - in case we will require to release a new patch to the migration.

Reply:
Done.